src/features/content-explorer/item-list/ItemList.js

```diff
@@ -231,6 +238,7 @@ const ItemList = ({
                 rowRenderer={renderRow}
                 noRowsRenderer={noItemsRenderer}
```

Comment:
have you tried using this `noRowsRenderer` prop?

Reply:
We already use it for contentExplorer. https://github.com/box/box-ui-elements/blob/master/src/features/content-explorer/content-explorer/ContentExplorer.js#L529
But as seen in screenshot, header is displayed on top of contents of `noRowsRenderer`

Reply:
ahh gotcha!

maybe we could add a check to if the `if` statement that if `noRowsRenderer` is not passed, then display the table as normal. something like 

```
if (!noItemsRenderer || items.length > 0) {
    [...]
}
```
that way usages of `ItemList` that don't have a custom `noRowsRenderer` will continue to display the headers as normal

Reply:
Makes sense yes, added that